File: P009/main.java
import java.lang.*;
import java.math.*;

public class main {
    private static void findProduct() {
        int sum = 1000;

        for(int a=1; a<sum/3; a++) {
            for(int b=a+1; b<sum/2; b++) {
                int c = sum-a-b;
                if(a*a + b*b == c*c) {
                    System.out.println(a*b*c);
                }
            }
        }
    }

    public static void main(String[] args) {
        long startTime = System.currentTimeMillis();

        findProduct();

        long endTime = System.currentTimeMillis();
        long totalSpan = System.currentTimeMillis();
        System.out.println("Completed in " + totalSpan + " milliseconds.");
    }
}

File: P009/main.py
import time
import math

def findProduct() :
    a = 0
    sum = 1000

    for a in range(1, int(sum/3)) :
        b = a+1
        while b < int(sum/2) :
            c = sum-a-b
            if a*a + b*b == c*c :
                return a*b*c
            b+=1
    return -1

start = time.time()

result = findProduct()
print result



elapsed = time.time()

print("found in %s seconds") % (elapsed)
